src/main.cpp

```diff
@@ -8904,7 +8894,8 @@ int64_t ComputeResearchAccrual(int64_t nTime, std::string cpid, std::string oper
     double ReferencePPD = dMagnitudeUnit*dAvgMag;
     if ((PPD > ReferencePPD*5))
     {
-            printf("Researcher PPD %f > Reference PPD %f for CPID %s with Lifetime Avg Mag of %f, Days %f \r\n",PPD,ReferencePPD,cpid.c_str(),dAvgMag,days);
+            printf("Researcher PPD %f > Reference PPD %f for CPID %s with Lifetime Avg Mag of %f, Days %f, MagUnit %f\r\n",
+                   PPD,ReferencePPD,cpid.c_str(),dAvgMag,days, dMagnitudeUnit);
```

Comment:
Will do. That one wasn't even supposed to be here but she can get a free ticket.